berkeley-mobile/StudyPact/CreatePreference/SelectClassView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  SelectClassView.swift
+//  berkeley-mobile
+//
+//  Created by Eashan Mathur on 1/30/21.
+//  Copyright © 2021 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+import SearchTextField
+
+class SelectClassView: UIView, UITextFieldDelegate, EnableNextDelegate {
+    let preferenceVC: CreatePreferenceViewController
```

Comment:
Someone correct me if I'm wrong on this, but this would create a retain cycle (`SelectClassView` has a strong reference to `preferenceVC`, which has a strong reference to `SelectClassView` through `preferenceVC.pages`). Could fix by making this `weak` or something similar.

Applies to `SelectClassView` and the other classes with strong references to `CreatePreferenceViewController`.